android/lib/build.gradle

```diff
@@ -16,7 +15,7 @@ android {
 }
 
 dependencies {
-  compile 'com.facebook.react:react-native:0.27.2'
+  compile 'com.facebook.react:react-native:+'
```

Comment:
For more context:
![screen shot 2016-07-08 at 3 43 01 pm](https://cloud.githubusercontent.com/assets/190648/16703462/aea98596-4522-11e6-9ff8-9b7c6dbe496f.png)